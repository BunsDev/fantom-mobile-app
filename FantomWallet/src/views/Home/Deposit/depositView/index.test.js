import React from 'react';
import { shallow } from 'enzyme';

// eslint-disable-next-line import/first
import DepositNavigationBar from './index';

test('renders correctly', () => {
  const wrapper = shallow(<DepositNavigationBar renderToastNotification={() => {}} />);
  expect(wrapper).toMatchSnapshot();
});
