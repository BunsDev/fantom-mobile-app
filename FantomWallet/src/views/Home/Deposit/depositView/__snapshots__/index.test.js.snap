// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "flex-end",
    }
  }
>
  <Connect(DepositViewInfo)
    renderToastNotification={[Function]}
  />
</View>
`;
